### DISCUSSION TITLE
How do I enable bluetooth?

### DISCUSSION THREAD

**Discussion by andradei** [Jas-SinghFSU/HyprPanel#548] [Q&A] [answered]
Do I need to add a specific `exec-once` command on `hyprland.conf`?

These are the packages I have installed that I think are related to bluetooth stuff:

```
-------------------------
  From Arch repository
-------------------------
bluez 5.79-1
bluez-libs 5.79-1
bluez-utils 5.79-1
gnome-bluetooth-3.0 47.1-1

-------------------------
        From AUR
-------------------------
libastal-bluetooth-git r619.990f031-1
```

**Comment by Jas-SinghFSU** [comment, marked as answer]
If you have all the dependencies, make sure your bluetooth is enabled. Ex:
```
systemctl enable --now bluetooth.service
```
That is all and your bluetooth should work.

**Reply by andradei** [reply]
Interesting, I though that's what the hyprpanel bluetooth module would do when it toggle it on (or something similar).

**Reply by andradei** [reply]
I get it, I start the service, the module just toggles it.
Thanks for replying.
